Applications/SlicerApp/Testing/Python/SlicerRestoreSceneViewCrashIssue3445.py

```diff
@@ -9,7 +9,7 @@
 
 class SlicerRestoreSceneViewCrashIssue3445(ScriptedLoadableModule):
     """Uses ScriptedLoadableModule base class, available at:
-    https://github.com/Slicer/Slicer/blob/master/Base/Python/slicer/ScriptedLoadableModule.py
+    https://github.com/Slicer/Slicer/blob/mainase/Python/slicer/ScriptedLoadableModule.py
```

Comment:
Not a big issue, but there are 13 places in comments where "master/Base" was renamed to "mainase".

Reply:
Looks like a mistake I made during rebase. I can fix

Reply:
Update pushed to fix this.